source/Calamari/Calamari.csproj

```diff
@@ -8,9 +8,9 @@
     <TargetFramework>net452</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Calamari.AzureScripting" Version="14.0.0" />
-    <PackageReference Include="Calamari.Common" Version="19.4.6" />
-    <PackageReference Include="Calamari.Scripting" Version="14.0.1" />
+    <PackageReference Include="Calamari.AzureScripting" Version="14.0.1" />
+    <PackageReference Include="Calamari.Common" Version="20.3.6" />
```

Comment:
Does Calamari follow SemVer? If so, this bump contains breaking changes. I assume we are happy that things are ok in case tests don't capture everything 😄. Also, it looks like version 20.3.8 is available, so should this actually get the latest while we are updating?

Reply:
Actually it looks like a few of these updates are already behind the latest versions available so maybe update all if we can.

Reply:
My understanding is that we bump the major version of Calamari to match the server release. So it's 20.* to go into master for the 2021.3 release of Octopus Server. So I don't think there's specifically any breaking changes in here.

Reply:
>  already behind the latest versions
That's why I shouldn't have left this PR sitting here for so long. I'll update to the latest

Reply:
Ok, that makes sense. If that is the case and we don't want to target the latest NuGet I am happy for to be merged 😄

Reply:
I've update to get the latest from nuget :)